metadata.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "puppetlabs-nodejs",
+  "name": "puppet-nodejs",
   "version": "0.8.0",
   "author": "puppetlabs",
```

Comment:
@nibalizer the namespace on the forge is `puppet`, not `puppetcommunity`, so I think `puppet` is fine